### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by guyiac** [skyslasher/wp-civi-mosaico#1]
What's the status of this project? It looks like a great idea - are you still maintaining it? Are you looking for help?

**Comment by skyslasher** [comment]
Hello,
I am still maintaining it, it is run productively for years on one of my WP/CiviCRM sites. Help is always appreciated. E.g. to get it in the official CiviCRM plugin catalogue.
Regards
Andreas

**Comment by guyiac** [comment]
Cool, I am going to try it out on a site. It looks nice. 

There are two levels of listings for Civi, and they are based on level of review. 

- The first is getting listed on the CiviCRM Extensions page. 
- The second is getting it listed in app (in CiviCRM itself) under the Add New tab at Administer --> System Settings --> Extensions.  

To get it reviewed, you need to get a trusted CiviCRM community member to review it. Though I am not a dev, I've been involved in the community a long time and could introduce you to a couple of devs who might be willing to review it if you don't know anyone. And jusdt confirming, this is a CiviCRM Extension, like Flexmailer, correct? Not a WordPress Plugin?

**Comment by skyslasher** [comment]
Correct, it is a CiviCRM Extension. It once started as a WordPress plugin, and I reviewed it and splitted it in three different CiviCRM Extensions (see [README.md](https://github.com/skyslasher/wp-civi-mosaico)). This plugin depends on one of these three plugins, [CiviCRM Mosaico Plugin Interface](https://github.com/skyslasher/de.ergomation.civi-mosaico-plugininterface) as mentioned in the README.md.
Of course, this Extension also depends on WordPress, but the other two do not.

**Comment by composerjk** [comment]
@skyslasher Since the [Mosaico extension](https://github.com/veda-consulting-company/uk.co.vedaconsulting.mosaico) has been updated to Mosaico 3.x, your 3 plugins/extensions might need an update. For the CiviCRM extension, it's probably also worth updated with the latest version of civix.

Saw reports that your plugins were no longer working in CiviCRM 5.69.5 ESR with WordPress 6.4.3. Have not had time to investigate.
